src/widgets/passwordless/passwordlessWidget.jsx

```diff
@@ -60,24 +66,24 @@ class MainView extends React.Component {
                 <Separator text={i18n('or')} />
             )}
             {isEmail && showIntro && <Intro>{i18n('passwordless.intro')}</Intro>}
-            {isEmail && <EmailInputForm handler={this.handleSubmit} onSuccess={this.handleSuccess} />}
+            {isEmail && <EmailInputForm handler={(data) => ReCaptcha.handle(data, this.props, this.callback, "passwordless_email")} onSuccess={this.handleSuccess} />}
             {!isEmail && showIntro && <Intro>{i18n('passwordless.sms.intro')}</Intro>}
-            {!isEmail && <PhoneNumberInputForm handler={this.handleSubmit} onSuccess={this.handleSuccess} />}
+            {!isEmail && <PhoneNumberInputForm handler={(data) => ReCaptcha.handle(data, this.props, this.callback, "passwordless_phone")} onSuccess={this.handleSuccess} />}
         </div>;
     }
 }
 
 class VerificationCodeView extends React.Component {
     handleSubmit = data => {
         const { apiClient, auth, phoneNumber } = this.props;
-
+        console.log(data)
```

Comment:
Beware with this 👁️

Reply:
This is really curious because I committed a new version removing the console.log a few minutes after. I double checked and it has indeed been removed.